server/container/spring/src/main/java/org/apache/james/container/spring/bean/factorypostprocessor/MailboxConfigurationBeanFactoryPostProcessor.java

```diff
@@ -36,11 +41,21 @@
  * It will register it with the alias mailboxmanager
  */
 public class MailboxConfigurationBeanFactoryPostProcessor implements BeanFactoryPostProcessor {
-
     /**
      * @see org.springframework.beans.factory.config.BeanFactoryPostProcessor#postProcessBeanFactory
      * (org.springframework.beans.factory.config.ConfigurableListableBeanFactory)
      */
+
+    private static final String JPA_MAILBOXMANAGER = "jpa-mailboxmanager";
+    private static final String MEMORY_MAILBOX_MANAGER = "memory-mailboxManager";
+    private static final String JCR_MAILBOXMANAGER = "jcr-mailboxmanager";
+    private static final String MAILDIR_MAILBOXMANAGER = "maildir-mailboxmanager";
+    private static final String HBASE_MAILBOXMANAGER = "hbase-mailboxmanager";
+    private static final String CASSANDRA_MAILBOXMANAGER = "cassandra-mailboxmanager";
+    private Set<String> mailboxManagerIds = new HashSet<String>(Arrays.asList(JPA_MAILBOXMANAGER, MEMORY_MAILBOX_MANAGER,
```

Comment:
This is not OK as an answer. See detailed  proposal for making it static final private immutable stuff :-)